From 80fb508ae6939cb43426b8a995de7520bf06038c Mon Sep 17 00:00:00 2001
From: "David.Wu" <david.wu@rock-chips.com>
Date: Tue, 4 Dec 2018 19:30:25 +0800
Subject: [PATCH] arm64: configs: Enable CONFIG_USB and CONFIG_USB_DWC3_GADGET
 for rk3399pro_npu

The driver of dwc3-rockchip depends on CONFIG_USB and
CONFIG_USB_DWC3_GADGE. To use the driver, need to enable
them.

Change-Id: Iddfbf1ac2b413aa33f5803ffcfc165ec3555937f
Signed-off-by: David Wu <david.wu@rock-chips.com>
---
 arch/arm64/configs/rk3399pro_npu_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/rk3399pro_npu_defconfig b/arch/arm64/configs/rk3399pro_npu_defconfig
index 107eab746354..93f4c7a00bcc 100644
--- a/arch/arm64/configs/rk3399pro_npu_defconfig
+++ b/arch/arm64/configs/rk3399pro_npu_defconfig
@@ -99,7 +99,9 @@ CONFIG_REGULATOR_FIXED_VOLTAGE=y
 CONFIG_REGULATOR_FAN53555=y
 CONFIG_REGULATOR_SYR82X=y
 # CONFIG_HID_GENERIC is not set
+CONFIG_USB=y
 CONFIG_USB_DWC3=y
+CONFIG_USB_DWC3_GADGET=y
 CONFIG_USB_GADGET=y
 CONFIG_USB_CONFIGFS=y
 CONFIG_USB_CONFIGFS_F_FS=y
-- 
2.35.3

